mongodump/mongodump_test.go

```diff
@@ -348,6 +348,33 @@ func setUpDBView(dbName string, colName string) error {
 	return nil
 }
 
+func setUpColumnstoreIndex(dbName string, colName string) error {
+	sessionProvider, _, err := testutil.GetBareSessionProvider()
+	if err != nil {
+		return err
+	}
+
+	createIndexCmd := bson.D{
+		{"createIndexes", colName},
+		{"indexes", bson.A{
+			bson.D{
+				{"key", bson.D{{"$**", "columnstore"}}},
+				{"name", "dump_columnstore_test"},
+				{"columnstoreProjection", bson.D{
+					{`"`, 0},
```

Comment:
Is this really legal syntax for this? That's really odd. Even if it's legal, can we instead do `{ "_id", 1 }` so it's less surprising?